app/models/row.js

```diff
@@ -2,81 +2,318 @@ import { alias } from '@ember/object/computed';
 import DS from 'ember-data';
 import { computed } from '@ember/object';
 
+/**
+ * The Row model represents a row in a data profile. It contains all the data necessary
+ * to display a row in a table.
+ *
+ * See Ember Data documentation: https://api.emberjs.com/ember-data/release/classes/DS.Model
+ */
 export default DS.Model.extend({
-  // number of geoids
+  /* =============================================
+  =            Row MetaData                      =
+  ============================================= */
+
+  /**
+   * Number of geoids, which compose an aggregate geography
+   */
   numGeoids: DS.attr('number'),
 
-  // categorical information
-  profile: DS.attr('string'), // e.g. demographic, social, housing, economic, census
-  category: DS.attr('string'), // e.g. sexAndAge (as part of profile "demographic"), placeOfBirth (as part of profile "social")
+  /**
+   * Profile Type, e.g. demographic, social, housing, economic, census
+   */
+  profile: DS.attr('string'),
+
+  /**
+   * Category name, used to group together rows
+   * e.g. sexAndAge (as part of profile "demographic"), placeOfBirth (as part of profile "social")
+   */
+  category: DS.attr('string'),
+
+  /**
+   * Raw variable name, used to group together rows.
+   * e.g. sexAndAge (as part of profile "demographic"), placeOfBirth (as part of profile "social")
+   */
   variable: DS.attr('string'), // e.g. Pop10t14 (as part of category "sexAndAge")
+
+  /**
+   * Lower-cased version of "variable"
```

Comment:
opposite -- `variablename` is the regular cased version, `variable` is the lowercase version